pkg/eraser/eraser.yaml

```diff
@@ -16,6 +16,7 @@ spec:
     ports:
     - containerPort: 80
     imagePullPolicy: Always
+    args:  ["--runtime=containerd", "imagelist=imagelist-sample"]
```

Comment:
why is this hardcoded?

Reply:
the eraser.yaml isn't used in the project, it was just for testing deploying the pods manually

Reply:
aren't we automatically setting runtime value

Reply:
yeah but only when imagejob schedules the pods . When scheduling the pods manually for testing purposes we have to provide it